### PR TITLE
Added timeout attribute for travis

### PR DESCRIPTION
**Pull request by humanjas**
#159

# Introduction
Fixed the timeout of the travis jobs during the build

# Purpose
To make sure that the travis build will timeout less likely upon building

# Details
none

# Future Applicability
-

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -88,8 +86,6 @@ install:
   - ./gradlew clean
   # Check components status
   - sdkmanager --list || true
-  # Check style
-  # - ./gradlew checkstyle
```
Comment (ashay123) [inline .travis.yml]: Why is this removed?

**Comment by ashay123** [comment]
I guess the changes that you've made didn't have the expected results.

File: .travis.yml (hunk 1 above)
Comment (humanjas) [inline .travis.yml]: Checkstyle is included in the check command

**Comment by humanjas** [comment]
@ashay123 Travis only builds upon a pr or an update of it. So that's why I am testing it in here.

**Comment by ashay123** [comment]
Alright! Successful build!
